### ISSUE TITLE
Modifying component/default subtype creates page/default components

### ISSUE THREAD

**Issue by PeterBowater** [blueriver/MuraCMS#2568]
As above, I created a class extension and a set of attributes for component/default, however when creating a new component/default it saved the component as subtype page/default, but showed it in the Components tab in the content view.

Very odd behaviour :)

I tried creating a new subtype component/xxxx with the same set of extended attributes, and that works absolutely fine, so just something going on with the default subtype not allowing it to be modified.

TVM,

Peter.

**Comment by mattlevine** [comment]
I can't seem to reproduce this. Are you able to repeat the steps and see the error happen again?

**Comment by PeterBowater** [comment]
Thanks for the reply Matt, I've just tried replicating (again, I did before posting), with no luck on latest version.

Procedure was:

Define a component/default extension in config.xml.cfm with one extended attribute
Reload Mura
Add a new component, fill in data, publish.
Component appeared in 'Components' tab in content view, with a 'Page' icon.

Odd... definitely had it happen more than once, will keep an eye out for it again, and report back if still a problem.

Sorry for the wild goose chase!

Peter.
